pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -74,6 +77,19 @@ func (a Adapter) Push(parameters common.PushParameters) (err error) {
 	s.End(true)
 
 	log.Infof("\nCreating Kubernetes resources for component %s", a.ComponentName)
+
```

Comment:
Just thinking, do we really need make run command mandatory, if `debug` is present?

Reply:
Keeping the debugger open might cause problems with security and other things.